src/EnergyPlus/SolarShading.cc

```diff
@@ -8404,15 +8404,19 @@ void CalcInteriorSolarDistribution(EnergyPlusData &state)
                         int PipeNum = state.dataSurface->SurfWinTDDPipeNum(SurfNum);
                         int SurfNum2 = state.dataDaylightingDevicesData->TDDPipe(PipeNum).Dome;
                         Real64 CosInc = state.dataHeatBal->SurfCosIncAng(state.dataGlobal->HourOfDay, state.dataGlobal->TimeStep, SurfNum2);
+                        Real64 GndReflSolarRad = state.dataEnvrn->GndSolarRad;
+                        if (state.dataSurface->UseSurfPropertyGndSurfRefl(SurfNum2)) {
+                            GndReflSolarRad = state.dataSurface->GndReflSolarRad(SurfNum2);
+                        }
```

Comment:
Sets the ground surfaces reflected solar radiation for each exterior surface if specified, instead of using the global ground reflected solar radiation.